js/utility/calculateDuration.js

```diff
@@ -0,0 +1,10 @@
+import { msPerDay } from "/js/constants.js";
+export default function calculateDuration(taskStartDateString) {
+  const startDate = new Date(taskStartDateString).getTime();
```

Comment:
see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse

```suggestion
  const startDate = Date.parse(taskStartDateString);
```

Reply:
done!